R/html_resources.R

```diff
@@ -70,17 +69,23 @@ find_external_resources <- function(input_file,
     path = character(0),
     explicit = logical(0),
     web = logical(0))
+
   input_dir <- dirname(normalize_path(input_file))
 
   # discover a single resource--tests a string to see if it corresponds to a
   # resource on disk; if so, adds it to the list of known resources and returns
   # TRUE
-  discover_single_resource <- function(path, explicit, web) {
```

Comment:
Personally I'm fine with the original style, since the list of arguments were short enough. The new style leaves a large amount of white spaces on the left. Usually I don't prefer wasting spaces like that if possible. My coding style is often very dense, so I can read most code without moving my eyes over a long distance.

Again, no strong opinion. I never really argue about coding styles -- use whatever style that makes you feel most comfortable if you are the main person to maintain the code.

Reply:
I might actually revert these types of line breaks for functions with low amounts of args in internal functions. I agree it's probably counter productive.